scan-payment/src/main/java/com/getbouncer/scan/payment/analyzer/NameDetectAnalyzer.kt

```diff
@@ -73,7 +105,31 @@ class NameDetectAnalyzer private constructor(
         )
     }
 
-    private suspend fun processPredictions(
+    private suspend fun processExpiryPrediction(
+        expRect: RectF,
+        bitmapForObjectDetection: Bitmap
+    ): ExpiryDetect.Expiry? {
+        // Crop out the expected aspect ratio
+        val targetAspectRatio = ASPECT_RATIO
+        val scaledRect = expRect.scaled(bitmapForObjectDetection.size())
+        val scaledExpRectNewHeight = scaledRect.width() * targetAspectRatio
+
+        val rect = Rect(
+            scaledRect.left.toInt(),
+            (scaledRect.centerY() - scaledExpRectNewHeight / 2).toInt(),
+            scaledRect.right.toInt(),
+            (scaledRect.centerY() + scaledExpRectNewHeight / 2).toInt()
+        )
```

Comment:
this should move into the `ExpiryDetect` logic. I want the calls to our analyzers to be as simple as possible with the least data massaging required possible.